frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Tab-Sidebar-ODC-4116.feature

```diff
@@ -0,0 +1,134 @@
+Feature: Monitoring tab on the Sidebar and Health Checks
+    As a user, I should be able to see Monitoring tab on the sidebar and add Health Checks
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke, @regression
+Scenario: Monitoring tab on the Sidebar for Deployments: MH-03-TC02
+    Given user is at Topology page
+    And user has a workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Monitoring tab
+    And user clicks on View Monitoring dashborad link
+    Then user will be taken to Dashboard tab on the Monitoring page
+
+
+@smoke, @regression
+Scenario: Opening Monitoring Dashboard link on the Sidebar for Deployments: MH-03-TC01
+    Given user is at Topology page
+    And user has a workload of Deployment type resource
+    When user clicks on the workload to open the sidebar
+    And user clicks on Monitoring tab
+    Then user will see View Monitoring dashborad link
+    And user will see CPU Usage Metrics
+    And user will see Memory Usage Metrics
+    And user will see Receive Bandwidth Metrics
+    And user will see All Events dropdown
+
+
+@smoke, @regression
+Scenario: Health Checks option in Advanced Options: MH-04-TC01
+    Given user is at +Add page
+    When user clicks on From Git card
+    Then user will see Health Checks advanced option
+
+
+# Git URL: https://github.com/openshift-roadshow/nationalparks-py
+@smoke, @regression
+Scenario: Health Checks option in Advanced Options: MH-04-TC02
+    Given user is at +Add page
+    When user clicks on From Git card
+    And user enters Git URL
+    And user selects Python Builder Image
+    And user enters name of the application as "national-parks-demo"
+    And user selects Deployment as a resource type
+    And user clicks on Health Checks advanced option
+    And user clicks on Add Readiness Probe
+    And user clicks on right button
```

Comment:
user clicks on tick icon sounds better, WDYT?

Reply:
updated the statements